server/app/controllers/api/ProgramApplicationsApiController.java

```diff
@@ -91,7 +91,7 @@ public CompletionStage<Result> list(
             .orElse(new IdentifierBasedPaginationSpec<>(pageSize, Long.MAX_VALUE));
 
     return programService
```

Comment:
Not blocking: Semi-unrelated to this change, but part of a pet project of mine. Assuming it doesn't cause any compile errors, mind marking this class as final to make the choice for subclassing explicit rather than implicit?

Reply:
done